Source/utils/display.cpp

```diff
@@ -160,8 +160,8 @@ bool SpawnWindow(const char *lpWindowName)
 #endif
     
 #if SDL_VERSION_ATLEAST(2, 0, 6) && defined(__IPHONEOS__)
-    SDL_SetHint(SDL_HINT_TOUCH_MOUSE_EVENTS, "0");
-    SDL_SetHint(SDL_HINT_MOUSE_TOUCH_EVENTS, "0");
+	SDL_SetHint(SDL_HINT_TOUCH_MOUSE_EVENTS, "0");
+	SDL_SetHint(SDL_HINT_MOUSE_TOUCH_EVENTS, "0");

```

Comment:
@StephenCWills would it make sens to disable this in general? Since we specifically handle both I don't see a point in either.

Reply:
We don't use mouse touch events at all. Honestly, I wasn't even sure what context they'd be generated in so maybe the hint is off by default on most platforms.

However, touch mouse events are used in diabloui. I'd expect that you'd be unable to navigate the menus with the touchscreen on iOS after making this change. Can someone check that?

Reply:
I tested that. I can confirm that you can still navigate the menus with the touchscreen

Reply:
I tested this on Android without the check for `defined(__IPHONEOS__)`. I am completely unable to skip videos and the splash screen. I am also unable to navigate menus. I wonder how this could work on iOS but not Android.

Reply:
My bad, the SDL_SetHint(SDL_HINT_TOUCH_MOUSE_EVENTS, "0"); is not needed

Reply:
The iPad mouse is really odd, it's works more like a touch emulator then a mouse, but it does have a movable pointer.

Reply:
SDL_SetHint(SDL_HINT_MOUSE_TOUCH_EVENTS, "0"); is needed for the mouse click not to send both a mouse click event and a touch event.

Reply:
Indeed, without `SDL_SetHint(SDL_HINT_TOUCH_MOUSE_EVENTS, "0");`, everything works fine on Android. I'm not sure if Android was using touch mouse events in the first place, but my mouse still works as expected so it shouldn't hurt anything to use `SDL_SetHint(SDL_HINT_MOUSE_TOUCH_EVENTS, "0");` on all platforms.

Reply:
> Without this change, try leaving a shop using a mouse on android. You'll see you character move as well

To satisfy my own curiosity, I did this test on both Android and Windows. Android does indeed generate mouse touch events so it's good to disable this hint. Windows doesn't generate mouse touch events, but I think it's more correct to indiscriminately disable them anyway.